test/unit/services/test_catalogue_item.py

```diff
@@ -922,6 +925,101 @@ def test_update_with_nonexistent_catalogue_category_id(
     assert str(exc.value) == f"No catalogue category found with ID: {catalogue_category_id}"
 
 
+def test_update_with_existent_manufacturer_id(
+    test_helpers, catalogue_item_repository_mock, manufacturer_repository_mock, catalogue_item_service
+):
+    """
+    Test updating manufacturer id to an existing id
+    """
+    # pylint: disable=duplicate-code
+    catalogue_item_info = {
+        "id": str(ObjectId()),
+        "catalogue_category_id": str(ObjectId()),
+        "name": "Catalogue Item A",
+        "description": "This is Catalogue Item A",
+        "properties": [
+            {"name": "Property A", "value": 20, "unit": "mm"},
+            {"name": "Property B", "value": False},
+            {"name": "Property C", "value": "20x15x10", "unit": "cm"},
+        ],
+    }
+    # pylint: enable=duplicate-code
+    full_catalogue_item_info = {
+        **catalogue_item_info,
+        "manufacturer_id": str(ObjectId()),
+    }
+    catalogue_item = CatalogueItemOut(**full_catalogue_item_info)
+
+    # Mock `get` to return a catalogue item
+    test_helpers.mock_get(
+        catalogue_item_repository_mock,
+        CatalogueItemOut(
+            manufacturer_id=catalogue_item.manufacturer_id,
```

Comment:
If updating it to a different `manufacturer_id` then the catalogue item coming from the database should have a different `manufacturer_id` to the one in the updated `catalogue_item`.
```suggestion
            manufacturer_id=str(ObjectId()),
```